setup.py

```diff
@@ -420,6 +420,33 @@ def _get_package_path(package_name: str) -> Path:
     if arg == "rebuild" or arg == "build":
         arg = "build"  # rebuild is gone, make it build
         EMIT_BUILD_WARNING = True
+    if arg == "develop":
+        print(
+            "Redirecting 'python setup.py develop' to 'pip install -e . -v --no-build-isolation'",
```

Comment:
Should we print warning here:
```
print_box(
            """
            WARNING: Redirecting 'python setup.py develop' to 'pip install -e . -v --no-build-isolation'
            """
        )
```

Reply:
I would use print_box but it's defined further down in the file which would require me to move it which I feel like is a bit much for this PR so going to leave as is for now.